package-res/resources/web/pentaho/data/_filter/_Element.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 define([
-  "pentaho/lang/Base"
+  "../../lang/Base"
```

Comment:
I understand this is consistent with other cases in the code base, but I question if it should... Why not use the absolute ID?